Document Podman (#189)
cirruslabs/cirrus-cli: INSTALL.md

# Prequisites

> Since currently CLI runs all of it's tasks either in [Docker](https://www.docker.com/) or [Podman](https://podman.io/), make sure one of these is installed.

Reviewer: Maybe `via` and links to the sections??

```suggestion
Since currently CLI runs all of it's tasks either via [Docker](#docker) or [Podman](#podman), make sure one of these is installed.
```

## Docker

> Docker is available on a variety operating systems and distributions.

Reviewer: And link here 

```suggestion
[Docker](https://docker.com/) is available on a variety operating systems and distributions.
```

OS-specific instructions can be found here: https://docs.docker.com/get-docker/

## Podman

> Podman is generally only available on Linux.

Reviewer: And here
```suggestion
[Podman](https://podman.io/) is generally only available on Linux.
```

> Distribution-specific instructions can be found here: https://podman.io/getting-started/installation

Reviewer: Probably it's worth to add a note about how to switch backends if both Docker and Podman are installed on the host.

Author: Documented in https://github.com/cirruslabs/cirrus-cli/pull/189/commits/9bceefc8bdf6f69e898c0907559dd370adc1e7dd.
